rollup.config.js

```diff
@@ -1,85 +1,106 @@
+import commonjs from '@rollup/plugin-commonjs';
+import replace from '@rollup/plugin-replace';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
 import terser from '@rollup/plugin-terser';
 import url from '@rollup/plugin-url'
 import svg from '@svgr/rollup'
 import dts from 'rollup-plugin-dts'
 import esbuild from 'rollup-plugin-esbuild'
 
+import pkg from './package.json' assert { type: 'json' }
+const dependencies = Object.keys(pkg.dependencies)
+
+const banner = [
+    `/**`,
+    ` * ${pkg.name} - v${pkg.version}`,
+    ` * Compiled ${(new Date()).toUTCString().replace(/GMT/g, 'UTC')}`,
+    ` *`,
+    ` * Copyright (c) ReachFive.`,
+    ` *`,
+    ` * This source code is licensed under the MIT license found in the`,
+    ` * LICENSE file in the root directory of this source tree.`,
+    ` **/`,
+].join('\n');
+
+// Ignore Luxon library's circular dependencies
+function onWarn(message) {
+    if ( message.code === 'CIRCULAR_DEPENDENCY' ) return;
```

Comment:
If we get another circular dependency with a different cause, the warning will be hidden as well.

Reply:
En effet. Mais c'est un comportement que j'ai rétablis de la version précédente.
Dans l'idéal il faudrait même savoir d'où vient cette dépendance circulaire pour la xorriger au lieu de la masquer.
Enfin là l'idée c'etait surtout de restaurer la ma même config que avant.